design/scenarios.md

```diff
@@ -0,0 +1,73 @@
+# Working Scenarios of Kata Container
+
+- [CRI](#cri)
+- [OCI based CRI Runtimes](#oci-based-cri-runtimes)
+  - [PodSandbox in OCI based runtimes](#podsandbox-in-oci-based-runtimes)
+  - [Typical working pattern of CRI runtimes](#typical-working-pattern-of-cri-runtimes)
+- [Frakti CRI Runtime](#frakti-cri-runtime)
+  - [The APIs Frakti rely on](#the-apis-frakti-rely-on)
+  - [Working pattern of Frakti](#working-pattern-of-frakti)
+- [Working with Docker](#working-with-docker)
+
+## CRI
+
+The operations defined in CRI RuntimeService are quite straightforward. There are two parts (as shown in the figure): PodSandbox operations and Container Operations.
+
+![The CRI RuntimeService](cri-runtime-service.png)
+
+The CRI clients simply call PodSandbox Run to start a sandbox (Pod), and Create / Start / Stop / Remove containers inside it. The life cycle of containers are inside a sandbox life cycle.
+
+For kata container, the sandbox are implemented with a virtual machine (agent is its delegator), and the containers are created with OCI spec by the agent.
+
+## OCI based CRI Runtimes
+
+The OCI spec is not aware of PodSandbox, and it defines the [lifecycle of containers](https://github.com/opencontainers/runtime-spec/blob/master/runtime.md#lifecycle).
+
+### PodSandbox in OCI based runtimes
+
+The cri-containerd and CRI-O create a special container for PodSandbox. Once get an RunPodSandbox request, the cri-containerd (or CRI-O) will launch container created with a configured image, and use the container as the PodSandbox.
+
+By default, the pod container is created through the same OCI runtime, however, for Kata, this will be mapped to a virtual machine. That is, the kata-runtime will do a “cheat”, create a virtual machine sandbox if it found the request is to create a PodSandbox instead of a user container.
+
+### Typical working pattern of CRI runtimes
+
+![CRI Runtimes](cri-oci.png)
+
+As shown in the above figure, the current CRI runtimes run OCI (or, say runc) containers on a storage (mounted filesystem) in a namespace with the OCI spec.
+
+When the Kata runtime work with the above CRI runtimes, as a result, in the design of kata containers (and runV, clear containers) the detail of the runtime part is illustrated in the following figure:
+
+![Work with OCI](oci-kata.png)
+
+## Frakti CRI Runtime
+
+### The APIs Frakti rely on
```

Comment:
I'd rather this section be described as and describe the "CRI-friendly" level  API provided by Kata Containers for CRI runtimes rather than listing what Frakti currently uses.  

The contents of the resulting section will depend on the resolution of discussion around API, https://github.com/kata-containers/runtime/pull/32

Reply:
once we push the discussion in  kata-containers/runtime#32 , these content should be put in another document